lenovo/thinkpad/t490/default.nix

```diff
@@ -26,5 +25,7 @@
 
     LABEL="pulseaudio_end"
     '';
+    
+    throttled.enable = lib.mkDefault true;
```

Comment:
```suggestion
    services.throttled.enable = lib.mkDefault true;
```

Reply:
It's inside an enclosing `sevices = { ... }`.